core/src/main/java/com/linecorp/armeria/internal/server/annotation/AnnotatedServiceTypeUtil.java

```diff
@@ -115,5 +116,16 @@
                 "Can't convert '" + str + "' to type '" + clazz.getSimpleName() + "'.");
     }
 
+    private static Boolean parseBoolean(String s) {
+        switch (Ascii.toLowerCase(s)) {
+            case "true":
+                return true;
+            case "false":
+                return false;
```

Comment:
Couldn't we take other common boolean expressions such as 
- `1` and `0`
- `yes` and `no`
https://github.com/spring-projects/spring-framework/blob/master/spring-core/src/main/java/org/springframework/core/convert/support/StringToBooleanConverter.java#L38-L46

Reply:
How about `on` and `off` then?